!SESSION 2018-06-10 17:31:13.275 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.8.0_131
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Command-line arguments:  -os win32 -ws win32 -arch x86_64

!ENTRY org.eclipse.egit.ui 2 0 2018-06-10 17:31:45.302
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\tobi'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY org.eclipse.cdt.core 1 0 2018-06-10 17:34:06.184
!MESSAGE Indexed 'drivers_rs232_0' (0 sources, 0 headers) in 0,002 sec: 0 declarations; 0 references; 0 unresolved inclusions; 0 syntax errors; 0 unresolved names (0 %)

!ENTRY org.eclipse.cdt.core 1 0 2018-06-10 17:36:58.837
!MESSAGE Indexed 'drivers_rs232_0' (3 sources, 8 headers) in 0,112 sec: 344 declarations; 977 references; 0 unresolved inclusions; 17 syntax errors; 87 unresolved names (6,2 %)

!ENTRY org.eclipse.cdt.launch 4 150 2018-06-10 17:37:59.404
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2018-06-10 17:37:59.441
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
!SESSION 2018-06-12 15:01:06.737 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.8.0_131
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Command-line arguments:  -os win32 -ws win32 -arch x86_64

!ENTRY org.eclipse.egit.ui 2 0 2018-06-12 15:02:10.423
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\tobi'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.
!SESSION 2018-06-12 15:51:30.815 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.8.0_131
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Command-line arguments:  -os win32 -ws win32 -arch x86_64

!ENTRY org.eclipse.egit.ui 2 0 2018-06-12 15:53:58.068
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\tobi'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY org.eclipse.cdt.launch 4 150 2018-06-12 15:55:28.265
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2018-06-12 15:55:28.266
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.cdt.launch 4 150 2018-06-12 16:25:26.672
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2018-06-12 16:25:26.672
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.cdt.launch 4 150 2018-06-12 16:26:03.826
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2018-06-12 16:26:03.826
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.cdt.launch 4 150 2018-06-12 16:26:25.035
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2018-06-12 16:26:25.035
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.cdt.launch 4 150 2018-06-12 16:27:19.294
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2018-06-12 16:27:19.294
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.cdt.launch 4 150 2018-06-12 16:46:44.610
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2018-06-12 16:46:44.610
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.cdt.launch 4 150 2018-06-12 16:51:57.462
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2018-06-12 16:51:57.462
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.cdt.launch 4 150 2018-06-12 16:59:47.706
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2018-06-12 16:59:47.706
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.cdt.launch 4 150 2018-06-12 17:07:53.576
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2018-06-12 17:07:53.576
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.cdt.launch 4 150 2018-06-12 17:11:33.685
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2018-06-12 17:11:33.685
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.cdt.launch 4 150 2018-06-12 17:17:47.596
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2018-06-12 17:17:47.596
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:289)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
!SESSION 2018-07-07 10:09:16.361 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.8.0_131
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Command-line arguments:  -os win32 -ws win32 -arch x86_64

!ENTRY org.eclipse.egit.ui 2 0 2018-07-07 10:09:49.267
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\tobi'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY org.eclipse.epp.logging.aeri.ide 2 17 2018-07-07 10:09:49.609
!MESSAGE Server ‘org.eclipse.epp.logging.aeri.ide.server’ failed with exception: dev.eclipse.org. ; version: 2.0.4.v20170307-1435
!STACK 0
java.net.UnknownHostException: dev.eclipse.org
	at java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	at java.net.InetAddress$2.lookupAllHostAddr(Unknown Source)
	at java.net.InetAddress.getAddressesFromNameService(Unknown Source)
	at java.net.InetAddress.getAllByName0(Unknown Source)
	at java.net.InetAddress.getAllByName(Unknown Source)
	at java.net.InetAddress.getAllByName(Unknown Source)
	at org.apache.http.impl.conn.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:44)
	at org.apache.http.impl.conn.HttpClientConnectionOperator.connect(HttpClientConnectionOperator.java:102)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:319)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:363)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:219)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:195)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:86)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:108)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:57)
	at org.apache.http.client.fluent.Executor.execute(Executor.java:206)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.request(IO.java:206)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.refreshConfiguration(IO.java:68)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.ServerConnection.startUp(ServerConnection.java:124)
	at com.google.common.util.concurrent.AbstractIdleService$2$1.run(AbstractIdleService.java:54)
	at com.google.common.util.concurrent.Callables$3.run(Callables.java:93)
	at java.lang.Thread.run(Unknown Source)
!SESSION 2019-05-15 05:44:12.882 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.8.0_131
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Command-line arguments:  -os win32 -ws win32 -arch x86_64

!ENTRY org.eclipse.e4.ui.workbench 2 0 2019-05-15 05:44:45.519
!MESSAGE Removing part descriptor with the 'org.eclipse.linuxtools.dataviewers.charts.view' id and the 'Chart View' description. Points to the invalid 'bundleclass://org.eclipse.ui.workbench/org.eclipse.ui.internal.e4.compatibility.CompatibilityView' class.

!ENTRY org.eclipse.launchbar.core 2 0 2019-05-15 05:44:46.297
!MESSAGE Enablement expression is missing for descriptor type org.eclipse.cdt.debug.core.coreBuildDescriptorType

!ENTRY org.eclipse.launchbar.core 2 0 2019-05-15 05:44:46.301
!MESSAGE Enablement expression is missing for config provider for org.eclipse.cdt.debug.core.coreBuildDescriptorType

!ENTRY org.eclipse.launchbar.core 2 0 2019-05-15 05:44:46.302
!MESSAGE Enablement expression is missing for config provider for org.eclipse.cdt.debug.core.coreBuildDescriptorType

!ENTRY org.eclipse.egit.ui 2 0 2019-05-15 05:44:47.239
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\tobi'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY org.eclipse.e4.ui.workbench 2 0 2019-05-15 05:44:47.825
!MESSAGE Removing part descriptor with the 'org.eclipse.linuxtools.gcov.view' id and the 'gcov' description. Points to the invalid 'bundleclass://org.eclipse.ui.workbench/org.eclipse.ui.internal.e4.compatibility.CompatibilityView' class.

!ENTRY org.eclipse.e4.ui.workbench 2 0 2019-05-15 05:44:48.668
!MESSAGE Removing part descriptor with the 'org.eclipse.linuxtools.gprof.view' id and the 'gprof' description. Points to the invalid 'bundleclass://org.eclipse.ui.workbench/org.eclipse.ui.internal.e4.compatibility.CompatibilityView' class.

!ENTRY org.eclipse.e4.ui.workbench 2 0 2019-05-15 05:44:48.729
!MESSAGE Removing part descriptor with the 'org.eclipse.linuxtools.valgrind.ui.valgrindview' id and the 'Valgrind' description. Points to the invalid 'bundleclass://org.eclipse.ui.workbench/org.eclipse.ui.internal.e4.compatibility.CompatibilityView' class.

!ENTRY org.eclipse.e4.ui.workbench 2 0 2019-05-15 05:44:48.748
!MESSAGE Removing part descriptor with the 'org.eclipse.pde.runtime.LogView' id and the 'Error Log' description. Points to the invalid 'bundleclass://org.eclipse.ui.workbench/org.eclipse.ui.internal.e4.compatibility.CompatibilityView' class.

!ENTRY org.eclipse.cdt.core 1 0 2019-05-15 05:45:36.225
!MESSAGE Indexed 'drivers_rs232_0' (3 sources, 4 headers) in 0,23 sec: 104 declarations; 248 references; 0 unresolved inclusions; 0 syntax errors; 0 unresolved names (0 %)
!SESSION 2019-10-01 16:58:56.680 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.8.0_131
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Command-line arguments:  -os win32 -ws win32 -arch x86_64

!ENTRY org.eclipse.launchbar.core 2 0 2019-10-01 16:59:37.501
!MESSAGE Enablement expression is missing for descriptor type org.eclipse.cdt.debug.core.coreBuildDescriptorType

!ENTRY org.eclipse.launchbar.core 2 0 2019-10-01 16:59:37.518
!MESSAGE Enablement expression is missing for config provider for org.eclipse.cdt.debug.core.coreBuildDescriptorType

!ENTRY org.eclipse.launchbar.core 2 0 2019-10-01 16:59:37.701
!MESSAGE Enablement expression is missing for config provider for org.eclipse.cdt.debug.core.coreBuildDescriptorType

!ENTRY org.eclipse.egit.ui 2 0 2019-10-01 16:59:38.391
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\tobi'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY org.eclipse.epp.logging.aeri.ide 2 17 2019-10-01 16:59:40.223
!MESSAGE Server ‘org.eclipse.epp.logging.aeri.ide.server’ failed with exception: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $. ; version: 2.0.6.v20170906-1226
!STACK 0
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:224)
	at com.google.gson.Gson.fromJson(Gson.java:887)
	at com.google.gson.Gson.fromJson(Gson.java:852)
	at com.google.gson.Gson.fromJson(Gson.java:801)
	at org.eclipse.epp.internal.logging.aeri.ide.server.json.Json.deserialize(Json.java:91)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.refreshConfiguration(IO.java:66)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.ServerConnection.startUp(ServerConnection.java:124)
	at com.google.common.util.concurrent.AbstractIdleService$DelegateService$1.run(AbstractIdleService.java:62)
	at com.google.common.util.concurrent.Callables$4.run(Callables.java:122)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:385)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:213)
	... 9 more

!ENTRY org.eclipse.cdt.launch 4 150 2019-10-01 17:00:09.149
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2019-10-01 17:00:09.149
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)

!ENTRY org.eclipse.cdt.launch 4 150 2019-10-01 17:00:31.175
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2019-10-01 17:00:31.175
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)

!ENTRY org.eclipse.cdt.launch 4 104 2019-10-01 19:28:36.269
!MESSAGE Program file does not exist
!STACK 0
java.io.FileNotFoundException: D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf not found
	at org.eclipse.cdt.launch.AbstractCLaunchDelegate2.verifyProgramPath(AbstractCLaunchDelegate2.java:684)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.checkBinaryDetails(LocalRunLaunchDelegate.java:260)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:88)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
!SUBENTRY 1 org.eclipse.cdt.launch 4 104 2019-10-01 19:28:36.270
!MESSAGE D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf not found
!STACK 0
java.io.FileNotFoundException: D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf not found
	at org.eclipse.cdt.launch.AbstractCLaunchDelegate2.verifyProgramPath(AbstractCLaunchDelegate2.java:684)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.checkBinaryDetails(LocalRunLaunchDelegate.java:260)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:88)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)

!ENTRY org.eclipse.cdt.launch 4 150 2019-10-01 19:28:52.902
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2019-10-01 19:28:52.902
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)

!ENTRY org.eclipse.cdt.launch 4 150 2019-10-01 19:50:59.182
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2019-10-01 19:50:59.183
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)

!ENTRY org.eclipse.cdt.launch 4 150 2019-10-01 19:53:09.303
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2019-10-01 19:53:09.303
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)

!ENTRY org.eclipse.cdt.launch 4 104 2019-10-01 19:54:11.274
!MESSAGE Program file does not exist
!STACK 0
java.io.FileNotFoundException: D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf not found
	at org.eclipse.cdt.launch.AbstractCLaunchDelegate2.verifyProgramPath(AbstractCLaunchDelegate2.java:684)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.checkBinaryDetails(LocalRunLaunchDelegate.java:260)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:88)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
!SUBENTRY 1 org.eclipse.cdt.launch 4 104 2019-10-01 19:54:11.274
!MESSAGE D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf not found
!STACK 0
java.io.FileNotFoundException: D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf not found
	at org.eclipse.cdt.launch.AbstractCLaunchDelegate2.verifyProgramPath(AbstractCLaunchDelegate2.java:684)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.checkBinaryDetails(LocalRunLaunchDelegate.java:260)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:88)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)

!ENTRY org.eclipse.cdt.launch 4 104 2019-10-01 19:54:40.626
!MESSAGE Program file does not exist
!STACK 0
java.io.FileNotFoundException: D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf not found
	at org.eclipse.cdt.launch.AbstractCLaunchDelegate2.verifyProgramPath(AbstractCLaunchDelegate2.java:684)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.checkBinaryDetails(LocalRunLaunchDelegate.java:260)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:88)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
!SUBENTRY 1 org.eclipse.cdt.launch 4 104 2019-10-01 19:54:40.627
!MESSAGE D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf not found
!STACK 0
java.io.FileNotFoundException: D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf not found
	at org.eclipse.cdt.launch.AbstractCLaunchDelegate2.verifyProgramPath(AbstractCLaunchDelegate2.java:684)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.checkBinaryDetails(LocalRunLaunchDelegate.java:260)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:88)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)

!ENTRY org.eclipse.cdt.launch 4 150 2019-10-01 19:55:34.682
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2019-10-01 19:55:34.682
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
!SESSION 2019-10-01 20:26:43.388 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.8.0_131
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Command-line arguments:  -os win32 -ws win32 -arch x86_64

!ENTRY org.eclipse.launchbar.core 2 0 2019-10-01 20:26:57.201
!MESSAGE Enablement expression is missing for descriptor type org.eclipse.cdt.debug.core.coreBuildDescriptorType

!ENTRY org.eclipse.launchbar.core 2 0 2019-10-01 20:26:57.226
!MESSAGE Enablement expression is missing for config provider for org.eclipse.cdt.debug.core.coreBuildDescriptorType

!ENTRY org.eclipse.launchbar.core 2 0 2019-10-01 20:26:57.227
!MESSAGE Enablement expression is missing for config provider for org.eclipse.cdt.debug.core.coreBuildDescriptorType

!ENTRY org.eclipse.egit.ui 2 0 2019-10-01 20:26:57.546
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\tobi'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY org.eclipse.epp.logging.aeri.ide 2 17 2019-10-01 20:26:59.418
!MESSAGE Server ‘org.eclipse.epp.logging.aeri.ide.server’ failed with exception: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $. ; version: 2.0.6.v20170906-1226
!STACK 0
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:224)
	at com.google.gson.Gson.fromJson(Gson.java:887)
	at com.google.gson.Gson.fromJson(Gson.java:852)
	at com.google.gson.Gson.fromJson(Gson.java:801)
	at org.eclipse.epp.internal.logging.aeri.ide.server.json.Json.deserialize(Json.java:91)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.refreshConfiguration(IO.java:66)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.ServerConnection.startUp(ServerConnection.java:124)
	at com.google.common.util.concurrent.AbstractIdleService$DelegateService$1.run(AbstractIdleService.java:62)
	at com.google.common.util.concurrent.Callables$4.run(Callables.java:122)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:385)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:213)
	... 9 more
!SESSION 2019-10-03 10:54:07.000 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.8.0_131
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Command-line arguments:  -os win32 -ws win32 -arch x86_64

!ENTRY org.eclipse.launchbar.core 2 0 2019-10-03 10:54:25.731
!MESSAGE Enablement expression is missing for descriptor type org.eclipse.cdt.debug.core.coreBuildDescriptorType

!ENTRY org.eclipse.launchbar.core 2 0 2019-10-03 10:54:25.781
!MESSAGE Enablement expression is missing for config provider for org.eclipse.cdt.debug.core.coreBuildDescriptorType

!ENTRY org.eclipse.launchbar.core 2 0 2019-10-03 10:54:25.782
!MESSAGE Enablement expression is missing for config provider for org.eclipse.cdt.debug.core.coreBuildDescriptorType

!ENTRY org.eclipse.egit.ui 2 0 2019-10-03 10:54:26.101
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\tobi'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY org.eclipse.epp.logging.aeri.ide 2 17 2019-10-03 10:54:27.872
!MESSAGE Server ‘org.eclipse.epp.logging.aeri.ide.server’ failed with exception: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $. ; version: 2.0.6.v20170906-1226
!STACK 0
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:224)
	at com.google.gson.Gson.fromJson(Gson.java:887)
	at com.google.gson.Gson.fromJson(Gson.java:852)
	at com.google.gson.Gson.fromJson(Gson.java:801)
	at org.eclipse.epp.internal.logging.aeri.ide.server.json.Json.deserialize(Json.java:91)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.refreshConfiguration(IO.java:66)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.ServerConnection.startUp(ServerConnection.java:124)
	at com.google.common.util.concurrent.AbstractIdleService$DelegateService$1.run(AbstractIdleService.java:62)
	at com.google.common.util.concurrent.Callables$4.run(Callables.java:122)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:385)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:213)
	... 9 more

!ENTRY org.eclipse.cdt.launch 4 150 2019-10-03 11:00:45.217
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2019-10-03 11:00:45.217
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)

!ENTRY org.eclipse.cdt.launch 4 150 2019-10-03 11:04:08.183
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2019-10-03 11:04:08.183
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)

!ENTRY org.eclipse.cdt.launch 4 150 2019-10-03 11:14:10.337
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2019-10-03 11:14:10.338
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\work\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
!SESSION 2019-10-03 18:18:11.536 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.8.0_131
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Command-line arguments:  -os win32 -ws win32 -arch x86_64

!ENTRY org.eclipse.launchbar.core 2 0 2019-10-03 18:18:57.624
!MESSAGE Enablement expression is missing for descriptor type org.eclipse.cdt.debug.core.coreBuildDescriptorType

!ENTRY org.eclipse.launchbar.core 2 0 2019-10-03 18:18:57.696
!MESSAGE Enablement expression is missing for config provider for org.eclipse.cdt.debug.core.coreBuildDescriptorType

!ENTRY org.eclipse.launchbar.core 2 0 2019-10-03 18:18:57.697
!MESSAGE Enablement expression is missing for config provider for org.eclipse.cdt.debug.core.coreBuildDescriptorType

!ENTRY org.eclipse.egit.ui 2 0 2019-10-03 18:18:59.163
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\tobi'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY org.eclipse.epp.logging.aeri.ide 2 17 2019-10-03 18:19:02.052
!MESSAGE Server ‘org.eclipse.epp.logging.aeri.ide.server’ failed with exception: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $. ; version: 2.0.6.v20170906-1226
!STACK 0
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:224)
	at com.google.gson.Gson.fromJson(Gson.java:887)
	at com.google.gson.Gson.fromJson(Gson.java:852)
	at com.google.gson.Gson.fromJson(Gson.java:801)
	at org.eclipse.epp.internal.logging.aeri.ide.server.json.Json.deserialize(Json.java:91)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.refreshConfiguration(IO.java:66)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.ServerConnection.startUp(ServerConnection.java:124)
	at com.google.common.util.concurrent.AbstractIdleService$DelegateService$1.run(AbstractIdleService.java:62)
	at com.google.common.util.concurrent.Callables$4.run(Callables.java:122)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:385)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:213)
	... 9 more
!SESSION 2019-11-02 16:34:43.681 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.8.0_131
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Command-line arguments:  -os win32 -ws win32 -arch x86_64

!ENTRY org.eclipse.launchbar.core 2 0 2019-11-02 16:35:06.648
!MESSAGE Enablement expression is missing for descriptor type org.eclipse.cdt.debug.core.coreBuildDescriptorType

!ENTRY org.eclipse.launchbar.core 2 0 2019-11-02 16:35:06.715
!MESSAGE Enablement expression is missing for config provider for org.eclipse.cdt.debug.core.coreBuildDescriptorType

!ENTRY org.eclipse.launchbar.core 2 0 2019-11-02 16:35:06.716
!MESSAGE Enablement expression is missing for config provider for org.eclipse.cdt.debug.core.coreBuildDescriptorType

!ENTRY org.eclipse.egit.ui 2 0 2019-11-02 16:35:07.285
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\tobi'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY org.eclipse.epp.logging.aeri.ide 2 17 2019-11-02 16:35:08.698
!MESSAGE Server ‘org.eclipse.epp.logging.aeri.ide.server’ failed with exception: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $. ; version: 2.0.6.v20170906-1226
!STACK 0
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:224)
	at com.google.gson.Gson.fromJson(Gson.java:887)
	at com.google.gson.Gson.fromJson(Gson.java:852)
	at com.google.gson.Gson.fromJson(Gson.java:801)
	at org.eclipse.epp.internal.logging.aeri.ide.server.json.Json.deserialize(Json.java:91)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.refreshConfiguration(IO.java:66)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.ServerConnection.startUp(ServerConnection.java:124)
	at com.google.common.util.concurrent.AbstractIdleService$DelegateService$1.run(AbstractIdleService.java:62)
	at com.google.common.util.concurrent.Callables$4.run(Callables.java:122)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:385)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:213)
	... 9 more

!ENTRY org.eclipse.cdt.launch 4 150 2019-11-02 16:36:08.635
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\dev\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2019-11-02 16:36:08.635
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\dev\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\dev\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
!SESSION 2019-11-18 18:17:31.017 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.8.0_131
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Command-line arguments:  -os win32 -ws win32 -arch x86_64

!ENTRY org.eclipse.launchbar.core 2 0 2019-11-18 18:18:13.255
!MESSAGE Enablement expression is missing for descriptor type org.eclipse.cdt.debug.core.coreBuildDescriptorType

!ENTRY org.eclipse.launchbar.core 2 0 2019-11-18 18:18:13.274
!MESSAGE Enablement expression is missing for config provider for org.eclipse.cdt.debug.core.coreBuildDescriptorType

!ENTRY org.eclipse.launchbar.core 2 0 2019-11-18 18:18:13.276
!MESSAGE Enablement expression is missing for config provider for org.eclipse.cdt.debug.core.coreBuildDescriptorType

!ENTRY org.eclipse.egit.ui 2 0 2019-11-18 18:18:13.909
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\tobi'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY org.eclipse.epp.logging.aeri.ide 2 17 2019-11-18 18:18:15.402
!MESSAGE Server ‘org.eclipse.epp.logging.aeri.ide.server’ failed with exception: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $. ; version: 2.0.6.v20170906-1226
!STACK 0
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:224)
	at com.google.gson.Gson.fromJson(Gson.java:887)
	at com.google.gson.Gson.fromJson(Gson.java:852)
	at com.google.gson.Gson.fromJson(Gson.java:801)
	at org.eclipse.epp.internal.logging.aeri.ide.server.json.Json.deserialize(Json.java:91)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.refreshConfiguration(IO.java:66)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.ServerConnection.startUp(ServerConnection.java:124)
	at com.google.common.util.concurrent.AbstractIdleService$DelegateService$1.run(AbstractIdleService.java:62)
	at com.google.common.util.concurrent.Callables$4.run(Callables.java:122)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:385)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:213)
	... 9 more

!ENTRY org.eclipse.cdt.launch 4 150 2019-11-18 18:20:50.306
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\dev\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2019-11-18 18:20:50.306
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\dev\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\dev\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)

!ENTRY org.eclipse.cdt.launch 4 150 2019-11-18 18:23:31.474
!MESSAGE Error starting process.
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\dev\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
!SUBENTRY 1 org.eclipse.cdt.launch 4 150 2019-11-18 18:23:31.474
!MESSAGE Cannot run program "D:\EDAptix\projects\arduissimo\dev\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
!STACK 0
java.io.IOException: Cannot run program "D:\EDAptix\projects\arduissimo\dev\tests\drivers\drivers_rs232\ws_drivers_rs232\drivers_rs232_0\Release\drivers_rs232_0.elf": Launching failed
	at org.eclipse.cdt.utils.spawner.Spawner.exec(Spawner.java:349)
	at org.eclipse.cdt.utils.spawner.Spawner.<init>(Spawner.java:91)
	at org.eclipse.cdt.utils.spawner.ProcessFactory.exec(ProcessFactory.java:91)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.exec(LocalRunLaunchDelegate.java:283)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.runLocalApplication(LocalRunLaunchDelegate.java:108)
	at org.eclipse.cdt.launch.internal.LocalRunLaunchDelegate.launch(LocalRunLaunchDelegate.java:76)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
